content/pages/om-oss.mdx

```diff
@@ -1,10 +1,41 @@
+  
+![En frokost på 17.mai, innendørs i en leilighet. 2 personer ser på hverandre og ler, med et glass champagne i hånden. Første personen er i en fin dress, den andre er i samekofte. Det er salat og brødmat over bordet, med 17.mai pynt som små norske flagg](../images/17mai.jpeg)  
+![6 personer sitter på tepper på gresset ved drammenselva. Det er en piknikk, med mange fargefulle tepper, mat, drikke og en høytaler. Det er sol og godt sommervær, og nesten alle i bildet har på solbriller og en solhatt. De smiler og ser mot kameraet som tar dette bildet. Folk sitter nærme hverandre, noen med et glass i hånden. En rullestol står rett bak en person](../images/piknik.jpg)  
+
+  
+Queer Hangout er fylt med "fiksere", så vi har klart å gjøre mye med lite. Alt, fra denne websiden til rullestolrampen, er det noen i Queer Hangout som har laget på dugnad. Men det er mye vi ikke kan gjøre uten penger, og vi setter derfor stor pris på alle donasjoner! 
+  
+Her er våre små og større mål.  
+  
+**Sunn, hjemmelaget mat**  
+Skeive er mer utsatt for diskriminering i arbeidslivet, og har ofte dårligere råd enn resten av befolkningen. Vi tilbyr derfor et sunt gratis måltid på våre treff, som vi lager sammen som en sosial aktivitet. Dette er en av våre største utgifter. En donasjon på 150 kr. vil gi et måltid til alle på en Queer Hangout (gjennomsnittlig 20 personer). Man kan vippse oss på #818416, tusen takk!  
+  
+**Skeivt ball under pride**  
+Vi skal ha et skeivt ball under Drammen Pride! Det blir som ditt gamle ungdomskoleball, men nå får du være ditt skeive selv. Fylt med fargerik pynt, gammel partymusikk, dansegulv, karaoke og en morsom fotobooth. Vi ønsker å kunne investere i pynt og annet praktisk som vi kan bruke om og om igjen, istedenfor billig pynt vi må kaste når dagen er over.  
+  
+**Utleie av basseng eller badstue**  
+Vi har en stor andel transpersoner i Queer Hangout og de føler seg ikke trygge i garderober og svømmehaller. Mange opplever at uansett hvilket kjønn garderobe man velger, så er det noen som blir sint eller skremt. PKI (Norges pasientorganiasjon for kjønnsinkogruens) har tidligere leid en hel svømmehall eller badstue for bare transpersoner, og dette har vært en storsukksess! Vi ønsker å gjøre det samme i Drammen.  
+  
+**Naturgruppe**  
+Vi ønsker å lage en skeiv tur- og naturgruppe. Her ønsker vi å investere i en bålpanne, stormkjøkken, fiskestanger, lommekniver og annet man trenger for naturaktiviteter.  
+  
+**Kurs i tegnspråk**  
+Vi ønsker at så mange som mulig kan bli med, og vi vil invitere grupper som det sjeldent tilrettelegges for. En av disse gruppene er døve, så vi gjennomføre et kurs i tegnspråk for alle Queer Hangout medlemmer.  
+
+
+Du kan donere til oss på vipps på nummer #818416 eller overføre til oss på kontonummer 1506.92.76013. Du kan også maile oss på queerhangout@posteo.no om du har spørsmål. Tusen takk!  
```

Comment:
Denne delen bør ha en egen undertittel nivå 2. Dette er viktig for accessibility, så man enkelt kan navigere til forskjellige seksjoner.

Reply:
Bruk trykkbare lenker (e.g. `[queerhangout@posteo.no](mailto:queerhangout@posteo.no)`)